.circleci/config.yml

```diff
@@ -2,126 +2,13 @@ workflows:
   version: 2
   node-multi-build:
     jobs:
-      - eslint-v6
-      - eslint-v7
-      - ts-eslint-v4
-      - node-v14
-      - node-v16
       - lint
```

Comment:
If the lint job (which also runs on GitHub Actions) is the only remaining job, do we need the CircleCI config at all anymore?

Reply:
I stopped using CircleCI 😉